docs/conf.py

```diff
@@ -66,7 +66,7 @@
 
 # General information about the project.
 project = 'MicroPython'
-copyright = '2014-2019, Damien P. George, Paul Sokolovsky, and contributors'
+copyright = 'The MicroPython Documentation is Copyright © 2014-2019, Damien P. George, Paul Sokolovsky, and contributors.'
```

Comment:
Looking at http://docs.micropython.org/en/latest/, this would lead to rendered text like:

> © Copyright The MicroPython Documentation is Copyright © 2014-2019, Damien P. George, Paul Sokolovsky, and contributors.

Which is, well, not attentive approach to the matter.

Reply:
I was aware of the way it would be rendered - that's why I specifically chose the wording. Note the full stop to separate it from the "Last Modified ..." suffix.

Perhaps you could explain your concern and/or a suggestion to improve it rather than criticising my attention to detail? e.g. Maybe a dash at the start would work?

> © Copyright — The MicroPython Documentation is Copyright © 2014-2019, Damien P. George, Paul Sokolovsky, and contributors.

Reply:
> Perhaps you could explain your concern and/or a suggestion to improve it rather than criticising my attention to detail?

Put it simple, I would have appreciated if you postponed your ideas for "improvements" to licensing representation peculiarities until my request for copyright info addition was resolved. Then we would have been able to discuss your proposals without hurry and in details, in particular a view that representing project as a tangled mix of fragmented licensing is unlikely a good idea. So, maybe, just maybe, there's no need to make some of the change which you try to make.

"Would have", because it seems too late now, as that idea seems to align well with project owner's non-desire to make much simpler and clearer changes.

Reply:
The reason this change was included here is because it aims to clarify what the copyright notice in the docs refers to.  It's intended to apply just to the docs (see eg bdb0d22fe2619393e2d8cd591dd14cd4081f8fc5).  But looking at the rendered form of the docs it feels like it applies to the whole project.  Hence the change.